### ISSUE TITLE
Relation Not work has exception getRelationQuery incompatible

### ISSUE THREAD

**Issue by BehroozBvk** [jenssegers/laravel-mongodb#2406]
- Laravel-mongodb Version: 3.8.0
- PHP Version: 7.4.3
- Database Driver & Version: MongoDB server version: 5.0.7
- Lumen version: 8.3.4

### Steps to reproduce
1.This is my code
```@php
$bonus=\App\Models\Bonus::with('user_bonuses')->first();
```
2. Bonus model is:
```@php
class Bonus extends Model
{
    protected $connection = 'mongodb';
    public $timestamps = false;

    public function user_bonuses()
    {
        return $this->hasMany(UserBonus::class);
    }
}
```
3. UserBonus model is:
```@php
class UserBonus extends Model
{
    protected $connection = 'mongodb';
    protected $fillable = [
        'bonus_id',
        'user_id',
        'wallet_id',
        'scope',
        'amount',
        'status',
        'used_at',
    ];

    public function bonuses()
    {
        return $this->belongsTo(Bonus::class, 'id', 'bonus_id');
    }
}

```
### Expected behaviour
As you can see, the code is expected to receive a bonus with all its user bonuses, but I get an exception.

### Actual behaviour
But I have this exception:
`[ErrorException]
Declaration of Jenssegers\Mongodb\Relations\HasMany::getRelationQuery(Illuminate\Database\Eloquent\Builder $query, Illuminate\Database\Eloquent\Builder $parent, $columns = Array) should be compatible with Illuminate\Database\Eloquent\Relations\Relation::getRelationQuery()`

Please, help me.
Thank you.

@jenssegers

**Comment by SanaviaNicolas** [comment]
Hi @BehroozBvk, try with this library: [Laravel mongo auto sync](https://github.com/offline-agency/laravel-mongo-auto-sync).

**Comment by BehroozBvk** [comment]
> 

thanks @SanaviaNicolas . But I think this problem also needs to be solved.

**Comment by divine** [comment]
> thanks @SanaviaNicolas . But I think this problem also needs to be solved.

Update to the latest version. This was fixed in https://github.com/jenssegers/laravel-mongodb/pull/2263.



> Hi @BehroozBvk, try with this library: [Laravel mongo auto sync](https://github.com/offline-agency/laravel-mongo-auto-sync).

Stop spamming on unrelated issues.

**Comment by BehroozBvk** [comment]
> > thanks @SanaviaNicolas . But I think this problem also needs to be solved.
> 
> Update to the latest version. This was fixed in #2263.
> 
> > Hi @BehroozBvk, try with this library: [Laravel mongo auto sync](https://github.com/offline-agency/laravel-mongo-auto-sync).
> 
> Stop spamming on unrelated issues.

Thank you very so much. upgrated to 3.8.4 and solved the problem.
